sycl/plugins/level_zero/pi_level_zero.cpp

```diff
@@ -5663,9 +5767,27 @@ pi_result piextUSMHostAlloc(void **ResultPtr, pi_context Context,
   return PI_SUCCESS;
 }
 
-pi_result piextUSMFree(pi_context Context, void *Ptr) {
+static pi_result piextUSMFreeImpl(pi_context Context, void *Ptr) {
```

Comment:
please do not use "pi" or "piext" prefix for anything than is not from PI APO

Reply:
Fixed.